synopsys-task/src/synopsys-task/application-constant.ts

```diff
@@ -197,6 +198,11 @@ export const BRIDGE_COVERITY_INSTALL_DIRECTORY_KEY =
 export const COVERITY_INSTALL_DIRECTORY_KEY = "coverity_install_directory";
 export const COVERITY_INSTALL_DIRECTORY_KEY_CLASSIC_EDITOR =
   "bridge_coverity_install_directory";
+
+export const COVERITY_EXECUTION_PATH_KEY = "coverity_execution_path";
+export const COVERITY_EXECUTION_PATH_KEY_CLASSIC_EDITOR =
+  "bridge_coverity_execution_path";
```

Comment:
For classic editor we are no longer following the snake_case convention.
 Now we are following the camelCase convention. Can you apply the camelCase for all the new fields?